sdks/java/core/src/main/java/org/apache/beam/sdk/coders/LengthPrefixCoder.java

```diff
@@ -142,4 +125,14 @@ protected long getEncodedElementByteSize(T value, Context context) throws Except
   public boolean isRegisterByteSizeObserverCheap(@Nullable T value, Context context) {
     return valueCoder.isRegisterByteSizeObserverCheap(value, Context.OUTER);
   }
+
+  @Override
+  public String getEncodingId() {
+    return "";
+  }
+
+  @Override
+  public Collection<String> getAllowedEncodings() {
```

Comment:
Incidentally, `getEncodingId` should be removed. Possibly replaced by a URN, but that can live in the `CoderEncoder`. And `getAllowedEncodings` fails to address the "version N+2" problem effectively, so I think that design is DOA.